documentation/modules/managing/proc-setting-broker-limits.adoc

```diff
@@ -64,6 +64,9 @@ spec:
 NOTE: `minAvailableBytesPerVolume` and `minAvailableRatioPerVolume` are mutually exclusive.
 This means that only one of these parameters should be configured.
 
+NOTE: By default, the storage check interval is 60 seconds.
+This is reasonable, but it is possible to change that in `.spec.kafka.config` setting `client.quota.callback.static.storage.check-interval`.
```

Comment:
```suggestion
NOTE: Additional options can be configured in the `spec.kafka.config` section.
The full list of supported options can be found in <LinkToTheQuotasPluginRepo>.
```

Reply:
Or at least list the other relevant options as well -> the factor fallback and validity duration.